spec/user/user_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+RSpec.describe User, type: :model do
+  subject { User.new(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.') }
+  before { subject.save }
+
+  it 'name should be present' do
+    subject.name = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'posts_counter must be greater or equal to 0' do
+    subject.posts_counter = -5
+    expect(subject).to_not be_valid
+  end
+end
```

Comment:
-  **[OPTIONAL]** It is advisable to write tests that cover enough functionalities, like the `recent_3_posts` method.